acceptance/localcluster/localcluster.go

```diff
@@ -410,22 +417,18 @@ func (l *Cluster) Start() {
 	l.mu.Lock()
 	defer l.mu.Unlock()
 
+	l.monitorStopper = make(chan struct{})
+
 	l.runDockerSpy()
 	l.initCluster()
 	l.createCACert()
 	l.createNodeCerts()
 	l.createClientCerts()
-
-	l.monitorStopper = make(chan struct{})
```

Comment:
nevermind, gotcha. Catching potential shutdown of the roach nodes is what we start the event loop for. Updated.